kogito-benchmarks-framework/src/main/java/org/kie/kogito/benchmarks/framework/Commands.java

```diff
@@ -0,0 +1,484 @@
+package org.kie.kogito.benchmarks.framework;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.InetAddress;
+import java.net.InetSocketAddress;
+import java.net.Socket;
+import java.net.UnknownHostException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Scanner;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+import org.apache.commons.lang3.StringUtils;
+import org.assertj.core.api.Assertions;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static org.kie.kogito.benchmarks.framework.Logs.appendln;
+import static org.kie.kogito.benchmarks.framework.Logs.appendlnSection;
+
+public class Commands {
+    private static final Logger logger = LoggerFactory.getLogger(Commands.class);
+
+    public static final String BASE_DIR = getBaseDir();
+    public static final String APPS_DIR = getAppsDir();
+    public static final String MVNW = Commands.isThisWindows ? "mvnw.cmd" : "./mvnw";
+    public static final boolean isThisWindows = System.getProperty("os.name").matches(".*[Ww]indows.*");
+    private static final Pattern numPattern = Pattern.compile("[ \t]*[0-9]+[ \t]*");
+    private static final Pattern quarkusVersionPattern = Pattern.compile("[ \t]*<quarkus.version>([^<]*)</quarkus.version>.*");
+    private static final Pattern trailingSlash = Pattern.compile("/+$");
+
+    public static String getArtifactGeneBaseDir() {
```

Comment:
How many characters do we save by shortening the name here? I don't think it justifies the readability hit.

Reply:
This is not even needed when I think about it. It is a leftover from a Quarkus online generator test.